src/selfie_avoiding_obstacles/src/road_obstacle_detector.cpp

```diff
@@ -279,16 +286,22 @@ void Road_obstacle_detector::passive_timer_cb(const ros::TimerEvent &time)
 
 bool Road_obstacle_detector::switchToActive(std_srvs::Empty::Request &request, std_srvs::Empty::Response &response)
 {
+  if (status_ != PASSIVE)
+    return false;
```

Comment:
Czyli jak zrobimy reset_node gdy jesteśmy na active to nic się nie stanie?

Reply:
Tak, w końcu przełączanie na tryb aktywny powinien coś zmieniać tylko wtedy, gdy jesteśmy w trybie pasywnym.

Reply:
To musi być oddzielna funkcja do resetu w takim razie.
Gdy coś pójdzie nie tak to po przejęciu kontroli chcemy, żeby lane_controller zresetował swoje stany (jesteśmy na prawym pasie, wszystkie calculated_distance na false i wyłączyć kierunkowskazy)